cluster-operator/src/main/java/io/strimzi/operator/cluster/operator/resource/ZookeeperLeaderFinder.java

```diff
@@ -30,32 +26,27 @@
 import java.security.cert.CertificateException;
 import java.security.cert.CertificateFactory;
 import java.util.Base64;
-import java.util.List;
 import java.util.Map;
+import java.util.Set;
 import java.util.function.Supplier;
 import java.util.regex.Matcher;
 import java.util.regex.Pattern;
 
-import static java.lang.Integer.parseInt;
-
 /**
  * Helper class for finding the leader of a ZK cluster
  */
 public class ZookeeperLeaderFinder {
-
     private static final ReconciliationLogger LOGGER = ReconciliationLogger.create(ZookeeperLeaderFinder.class);
 
     private static final Pattern LEADER_MODE_PATTERN = Pattern.compile("^Mode: leader$", Pattern.MULTILINE);
 
-    public static final int UNKNOWN_LEADER = -1;
+    public static final String UNKNOWN_LEADER = "-1";
```

Comment:
Can you please explain, why is it better to operate with `String` instead of `int`?

Reply:
The methods don't return pod indexes from StatefulSet anymore but pod names => i.e. Strings. So they cannot return an integer when the leader is not found. So I made the `-1` a String as well so that it could be returned. I'm not sure if there is some better way to do this.